src/arch/arm/config.cmake

```diff
@@ -23,6 +23,11 @@ elseif(KernelArmCortexA53)
 elseif(KernelArmCortexA57)
     set(KernelArmPASizeBits44 ON)
     math(EXPR KernelPaddrUserTop "(1 << 44) - 1")
+elseif(KernelArmCortexA72)
+    # For Cortex-A72 in AArch64 state, the physical address range is 44 bits
+    # (https://developer.arm.com/documentation/100095/0001/memory-management-unit/about-the-mmu)
```

Comment:
maybe put the link to the docs to the commit messages instead of here?

Reply:
Actually, we have put them here on purpose, because in the end you are looking at the source and not at the commit message (or the commit/blame log) when you try to understand or debug something. We can also add it to the commit message, but I have some doubt about the practical value.

Reply:
The doc link here showed where you get the PA size. I agree docs in source code might help a lot, but this information is not very hard to get.

BTW, does the ID_AA64MMFR0_EL1 have 16T in the PARange field on real hardware (it probably should)? I know some customised SoCs cut on the actual PA size bits.

Reply:
So according to https://developer.arm.com/documentation/100095/0002/system-control/aarch64-register-descriptions/aarch64-memory-model-feature-register-0--el1, the PARange field "returns 0x4 to indicate that the processor supports a 44-bit physical address range, that is, 16TByte". I am not sure what is returned on the actual hardware though.